micro-rdk/src/common/entry.rs

```diff
@@ -100,7 +100,10 @@ pub async fn serve_web_inner<S>(
     let (cfg_response, cfg_received_datetime) = app_client
         .get_app_config(Some(network.get_ip()))
         .await
-        .unwrap();
+        .unwrap_or_else(|e| {
+            log::error!("failed to get valid app config: {}", e.to_string());
+            (Box::new(ConfigResponse::default()), None)
```

Comment:
invalid config response gets handled when matching on `RobotRepresentation`, using an empty robot.
verifying the behavior.